core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -2282,4 +2282,33 @@ public int getStaleJobThresholdInDays() {
         return Integer.parseInt(getOptional("kylin.tool.health-check.stale-job-threshold-days", "30"));
     }
 
+    /**
+     * get assigned server array, which a empty string array in default
+     * @return
+     */
+    public String[] getAssignedServers() {
+        return getOptionalStringArray("kylin.cube.schedule.assigned-servers", new String[] {});
+    }
+
+    /**
+     * Determine if the target node is in the assigned node
+     * @param targetServers target task servers
+     * @return
+     */
+    public Boolean isOnAssignedServer(String... targetServers) {
```

Comment:
The primitive type of JAVA is advised to use in Kylin

Reply:
I adjusted the config position and changed 'Boolean' to boolean just now, and finally i rebased this commit, could you review it again?:(